modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod to a node if the pod does not match the toleration. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the Machine Set manifest and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+The {product-title} supports using the AWS Load Balancer (ALB) Operator in an existing VPC with configured Local Zone subnets. This means that you can use the Operator to manage an `aws-load-balancer-controller` resource that discovers any Local Zone subnets defined in your existing VPC. You can then create an `Ingress` resource, so that you can expose a running cluster from one Local Zone subnet to other subnets that operate in the same VPC.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You configured an AWS Local Zones environment. 
+* You deployed your cluster om a compatible cloud platform. 
+* You installed the AWS Load Balancer (ALB) Operator.
+* You created an instance of the AWS Load Balancer Controller.
+
+.Procedure
+
+. Ensure that your worker manifest file for the Local Zone is configured to run workloads. The `zone_group` field specifies your AWS Local Zone that you selected on your AWS account.
++
+.Example of a configured portion of the worker manifest file that the VPC uses to run workloads
+[source,yaml]
+----
+...
+spec:
+  metadata:
+    labels:
+      zone_type: local-zone
+      zone_group: ${LZ_ZONE_NAME::-1}
+      node-role.kubernetes.io/edge: ""
+  taints:
```

Comment:
those labels were changed to: https://github.com/openshift/enhancements/pull/1232/files#diff-f2590494c9138551ab5b3d5632d11170fe8742a128256f9e1d8ebc63baa19f41R437-R440

Reply:
Thanks, updated.